doc/install-on-docker.md

```diff
@@ -68,7 +135,7 @@ See the [Hyperparameter optimization documentation](../examples/hyperopt/README.
 ### Batch L-BFGS
 
 ```
-cd ~/ray/examples/lbfgs/
+cd /ray/examples/lbfgs/
```

Comment:
I looked at forums a bit and these warnings seem to come from some recent changes in Tensorflow. I don't believe this is related to Ray or Docker and it something we may want to come back to.